tests/python/test_python_source_code.py

```diff
@@ -0,0 +1,81 @@
+from app.python.file_utils import *
+from app.python.PythonSourceCode import *
+from . import client
+import json
+
+# testing builder 
+def test_pythonsourcecode_builder_if(client): 
```

Comment:
Name should be tes_pythonsourcecode_first_builder and so on with the other test names

Reply:
@yeografik  ok, thanks for the recommendations, I will contemplate them!